src/components/application_manager/test/message_helper/message_helper_test.cc

```diff
@@ -359,10 +365,10 @@ TEST(MessageHelperTestCreate,
   EXPECT_CALL(*appSharedMock, app_id())
       .Times(AtLeast(5))
       .WillRepeatedly(Return(1u));
-  application_manager_test::MockApplicationManager mock_application_manager;
+  application_manager_test::MockApplicationManager mock_application_manager_;
   smart_objects::SmartObjectList ptr =
       MessageHelper::CreateAddVRCommandRequestFromChoiceToHMI(
-          appSharedMock, mock_application_manager);
+          appSharedMock, mock_application_manager_);
 
   EXPECT_FALSE(ptr.empty());
 
```

Comment:
In this case I had mistake and rename local variable according to code style for the class member variable.
This fixed in separate commit:https://github.com/smartdevicelink/sdl_core/pull/3259/commits/56a9fbc7389daa47d95b91e9f20807e33b20298a